packages/components/switch/src/Switch.doc.mdx

```diff
@@ -29,6 +29,12 @@ import { Switch } from "@spark-ui/switch"
 
 <Canvas of={stories.Default} />
 
+## With FormField
+
+If you need more fine-grained control over the state of the Switch (requirement/validation feedbacks, ...) you can also use it within a `FormField` component. you can refer to [dedicated documentation](./?path=/docs/experimental-formfield--docs) for more details.
```

Comment:
If you need to provide more context about the switch (for example, a helper message or a error message) the `FormField` component could be used. See  [documentation](./?path=/docs/experimental-formfield--docs) for more details.

Reply:
Done